DependencyInjection/CmfCreateExtension.php

```diff
@@ -142,6 +146,14 @@ public function loadPhpcr($config, XmlFileLoader $loader, ContainerBuilder $cont
         }
     }
 
+    public function loadOrm($config, XmlFileLoader $loader, ContainerBuilder $container)
+    {
+        $container->setParameter($this->getAlias() . '.backend_type_orm', true);
```

Comment:
i think the backend_type_X is a copy-paste thing from other bundles that provide model classes. we don't need this here and actually backend_type_phpcr could be dropped too